gufe/protocols/protocolunit.py

```diff
@@ -328,8 +329,8 @@ def execute(
                 end_time=datetime.datetime.now(),
             )
 
-        except KeyboardInterrupt:
-            # if we "fail" due to a KeyboardInterrupt, we always want to raise
+        except (KeyboardInterrupt, ExecutionInterrupt):
+            # we always want to raise in these situations
```

Comment:
@dotsdl - I'm pretty sure this is redundant, and that `KeyboardInterrupt` and `ExecutionInterrupt` get raised even if this catch isn't here.

Reply:
Ah, fair point. I think for the sake of understandability later (and seeing where e.g. `ExecutionInterrupt` actually gets used in the codebase), I'd rather leave it in.

Reply:
Works for me, I just added a comment so it's documented for clarity.